src/dgipy/network_graph.py

```diff
@@ -90,162 +108,25 @@ def create_network(
     :param search_mode: String indicating whether query was gene-focused or drug-focused
     :return: a networkx graph of drug-gene interactions
     """
-    interactions_graph = _initalize_network(interactions, terms, search_mode)
+    interactions_graph = initalize_network(interactions, terms, search_mode)
     _add_node_attributes(interactions_graph, search_mode)
```

Comment:
why the switch from private to public for `initialize_network`? (for my own understanding)

Reply:
Privatizing `initalize_network` in network_graph.py and referencing it in graph_app.py throws the ruff error:
 [](https://docs.astral.sh/ruff/rules/private-member-access/)